rest_framework_api_key/models.py

```diff
@@ -18,6 +19,27 @@ def create(self, *, secret_key=None, **kwargs):
         return super().create(**kwargs)
 
 
+class APIKeyGroup(models.Model):
+    name = models.CharField(
+        max_length=50,
+        unique=True,
+        help_text='The name of the API key group',
+    )
+
+    endpoint_permissions = models.ManyToManyField(
+        'EndpointPermission',
+        related_name='api_key_groups',
+        blank=True,
```

Comment:
Are there still default "public" endpoints that wouldn't require permissions perhaps? (I say "public" when I really mean "has an API key, but only access to default endpoints".)

Reply:
I see your point Guy. Strictly, I think this is hangover from what was *initially* a multi-stage migration process which *first* involved creating API Key Groups in the dev DB and *subsequently* involved migrating to API Key Groups with the option to add `endpoint_permissions`. I've since merged these migrations into one migration file.

We could do this with or without `blank=True`, but I feel that having `blank=True` provides more flexibility from the admin panel user's point of view because it gives them the ability to create API Keys and Group them without having to worry about endpoint permissions (ie they can be separated into two distinct processes if desired).

Reply:
And to follow up on @willcroft's point - yes - an application using this repository could set the default behaviour of an `APIKeyGroup` without `endpoint_permissions` to have access to a default set of endpoints as opposed to none at all. However, it might be better to just populate `APIKeyGroup`s with default `endpoint_permissions` upon creation instead?